src/status_im/contexts/chat/messenger/photo_selector/effects.cljs

```diff
@@ -80,9 +81,47 @@
                        (callback @albums))))))
               (callback @albums)))))))))
 
+(defn- get-album-cover-uri
+  [photos]
+  (get-in (first photos) [:node :image :uri]))
+
+(defn- get-album-info
+  [album]
+  ;; Easy promise chains without (.then #(...))
+  (p/->> (cameraroll/promise-get-photos
+          {:first      1
+           :groupTypes "albums"
+           :groupName  (:title album)
+           :assetType  "photos"})
+         :edges
+         (get-album-cover-uri)
+         (assoc album :uri)))
```

Comment:
`p/->` and `p/->>` allows to chain promises without `.then`, which is very similar to clojure's `->/->>`